.eslintrc.yml

```diff
@@ -0,0 +1,60 @@
+extends:
+- airbnb-base
+- next
+
+env:
+  browser: true
+  es2021: true
+  es6: true
+  jest: true
+
+globals:
+  google: readonly
+  module: readonly
+  process: readonly
+
+parserOptions:
+  ecmaVersion: 8
+  sourceType: module
+  ecmaFeatures:
+    jsx: true
+
+rules:
+  arrow-body-style: off
+  arrow-parens: off
+  func-names: warn
+  func-style: off
+  max-len: warn
+  no-bitwise: off
+  no-cond-assign:
+  - warn
+  - except-parens
+  no-confusing-arrow: off
+  no-console:
+  - warn
+  - allow:
+    - warn
+    - error
+  no-mixed-operators: off
+  no-nested-ternary: off
```

Comment:
Not much of a preference here. I only disabled [this rule](https://eslint.org/docs/rules/no-nested-ternary) because the example given (in that link) actually seems like it makes things worse, not to mention turns what should have been a `const` into a `let`.